openaz-xacml/src/main/java/org/apache/openaz/xacml/std/jaxp/JaxpAttributeAssignment.java

```diff
@@ -51,9 +51,9 @@
  */
 public class JaxpAttributeAssignment extends StdMutableAttributeAssignment {
 
-    protected JaxpAttributeAssignment(Identifier attributeIdIn, Identifier categoryIdIn, String issuerIn,
+    protected JaxpAttributeAssignment(Identifier categoryIdIn, Identifier attributeIdIn, String issuerIn,
                                       AttributeValue<?> attributeValueIn) {
-        super(attributeIdIn, categoryIdIn, issuerIn, attributeValueIn);
+        super(categoryIdIn, attributeIdIn, issuerIn, attributeValueIn);
```

Comment:
this looks like from documentation, `StdMutableAttributeAssignment(Identifier categoryIn, Identifier attributeIdIn, java.lang.String issuerIn, AttributeValue<?> attributeValueIn)`